Assets/MixedRealityToolkit.Tests/PlayModeTests/ManipulationHandlerTests.cs

```diff
@@ -672,18 +672,27 @@ private void RecordTransform(Transform transform, string description)
         public IEnumerator ManipulationHandlerOriginOffset()
         {
             TestUtilities.PlayspaceToOriginLookingForward();
+            // Without this background object that contains a collider, Unity will rarely
+            // return no colliders hit for raycasts and sphere casts, even if a ray / sphere is 
+            // intersecting the collider. Causes tests to be unreliable.
+            // It seems to only happen when one collider is in the scene.
```

Comment:
i had to read this several times to understand what's going on. can we rewrite this a bit like:

It seems to be a problem to have only one collider (testObject) in the scene. Unity will occasionally return an empty hit result even if there's an intersection with the testObject collider. To work around this issue we're adding a second gameobject with collider in the background.

Reply:
now that i read it after rewriting the old one also works for me - so ignore this if you want :D